modules/page_base.py

```diff
@@ -533,6 +533,51 @@ def context_click(
             el = self.fetch(reference, labels)
             self.actions.context_click(el).perform()
         return self
+    
+    def paste_to_element(
+        self, sys_platform, reference: Union[str, tuple, WebElement], labels=[]
+    ) -> Page:
+        """ Paste the copied item into the element"""
+        with self.driver.context(self.context_id):
+            el = self.fetch(reference, labels)
+            self.scroll_to_element(el)
+            self.actions.click(el).perform()
```

Comment:
Do you need to use ActionChains here? I think `el.click()` or `self.click_on(el)` would be fine